airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/components/StatusView.tsx

```diff
@@ -143,8 +170,15 @@ const StatusView: React.FC<StatusViewProps> = ({ connection }) => {
           </div>
         }
       >
-        {jobs.length ? <JobsList jobs={jobs} /> : <EmptyResource text={<FormattedMessage id="sources.noSync" />} />}
+        {jobs?.length ? <JobsList jobs={jobs} /> : <EmptyResource text={<FormattedMessage id="sources.noSync" />} />}
       </ContentCard>
+      <footer className={styles.footer}>
+        {(moreJobPagesAvailable || isJobPageLoading) && (
```

Comment:
:art: Given that the `footer` itself has some dimension (atm because of the `height`, if that's changed likely still because of the `margin`/`padding`), I'd suggest to also hide that if the button isn't shown, so we don't leave unnecessary "white-space" on the page.

Reply:
Done